app/views/internal/feedback_messages/_abuse_reports.html.erb

```diff
@@ -0,0 +1,69 @@
+<h2>Abuse Reports</h2>
+
+<div class="row my-3">
+  <div class="col">
+    <ul class="nav nav-pills">
+      <li class="nav-item">
+        <a href="/internal/reports?state=<%= @feedback_type %>&status=Open" class="nav-link <%= "active" if @status == "Open" %>">Open/Unresolved</a>
```

Comment:
How would you feel about using a path helper here, that's IMHO a bit nicer than building the url manually:

```ruby
internal_reports_path(state: 'foo', status: "Open")
#=> "/internal/reports?state=foo&status=Open"
```

Reply:
Sure. I was just moving this code, but I can go ahead and do that while I'm here. :smile:

Reply:
Yes, I realized afterward that you probably are just moving things around, but since we touch it already, we might as well clean it up a little bit.